internal/contour/cluster.go

```diff
@@ -27,6 +28,11 @@ const (
 	annotationMaxPendingRequests = "contour.heptio.com/max-pending-requests"
 	annotationMaxRequests        = "contour.heptio.com/max-requests"
 	annotationMaxRetries         = "contour.heptio.com/max-retries"
+	annotationUpstreamProtocol   = "contour.heptio.com/upstream-protocol"
+)
+
+const (
+	upstreamProtocolH2 = "h2"
```

Comment:
I don't think this needs a constant longer than it's value, we only use it in one place and it's not going to change so maybe just use the "h2" literal where needed

Reply:
i kept the constant because now i'm using it more than once. Tell me if you want me to remove the constant and hardcode them instead.

Reply:
please use "h2" upstreamProtocolh2 is longer without conveying more information. Or to say it another way, a constant should not encode it's value in its name.

Reply:
Done.